env/dev/clj/user.clj

```diff
@@ -1,21 +1,31 @@
 (ns user
+  "Userspace functions you can run by default in your local REPL."
   (:require
-    [sample-4.config :refer [env]]
+   [sample4.config :refer [env]]
+    [clojure.pprint]
     [clojure.spec.alpha :as s]
     [expound.alpha :as expound]
     [mount.core :as mount]
-    [sample-4.figwheel :refer [start-fw stop-fw cljs]]
-    [sample-4.core :refer [start-app]]))
+    [sample4.core :refer [start-app]]))
 
 (alter-var-root #'s/*explain-out* (constantly expound/printer))
 
-(defn start []
-  (mount/start-without #'sample-4.core/repl-server))
+(add-tap (bound-fn* clojure.pprint/pprint))
```

Comment:
we haven't used tap> before. probably need to research this a bit and add to the instructions